cdflib/cdfwrite.py

```diff
@@ -534,10 +534,19 @@ def close(self):
                 self._write_ccr(f, g, self.compression)
             if self.checksum:
                 if (self.compression > 0):
-                    g.seek(0, 2)
-                    g.write(self._md5_compute(g))
+                    md5_g = self._md5_compute(g)
+                    try:
+                        g.seek(0, 2)
+                        g.write(md5_g)
+                    except IOError as  e:
+                        print str(e)
```

Comment:
having the print statement like this would prevent this file being imported on Python 3. The best thing to do is convert this to the `print()` function and `from __future__ import print_function` at the very top of the file, this will then work under Python 2 and 3.